membrane_curvature/tests/test_mdakit_membcurv.py

```diff
@@ -155,33 +155,41 @@ def test_core_fast_leaflets():
         assert_almost_equal(z, z_test)
 
 
-@pytest.mark.parametrize('dummy_coordinates, test_mapper, n_cells, max_width', [(
-    # dummy coordinates (x,y)
-    ((0, 0), (1, 0), (2, 0),
-     (0, 1), (1, 1), (2, 1),
-     (0, 2), (1, 2), (2, 2)),
-    # should map to
-    lambda xy: (xy[0], xy[1]),
-    3, 3),
-])
-def test_grid_map_small_9grid(dummy_coordinates, test_mapper, n_cells, max_width):
-    factor = np.float32(n_cells / max_width)
-    for dummy_coord in dummy_coordinates:
-        assert test_mapper(dummy_coord) == grid_map(dummy_coord, factor)
-
-
-@pytest.mark.parametrize('dummy_coordinates, test_mapper, n_cells, max_width', [(
-    # dummy coordinates (x,y)
+@pytest.mark.parametrize('grid', [(
+    # dummy coordinates (x,y) in grid of 3x3
+    (0, 0), (1, 0), (2, 0),
+    (0, 1), (1, 1), (2, 1),
+    (0, 2), (1, 2), (2, 2)),
+    # dummy coordinates (x,y) in grid of 5x5
     ((0, 0), (1, 0), (2, 0), (3, 0), (4, 0),
      (0, 1), (1, 1), (2, 1), (3, 1), (4, 1),
      (0, 2), (1, 2), (2, 2), (3, 2), (4, 2),
      (0, 3), (1, 3), (2, 3), (3, 3), (4, 3),
-     (0, 4), (1, 4), (2, 4), (3, 4), (4, 4)),
+     (0, 4), (1, 4), (2, 4), (3, 4), (4, 4))])
+def test_grid_map_grids(grid):
+    # map to grid of 3 unit cells
+    factor_1 = 1
+    for dummy_coord in grid:
+        assert grid_map(dummy_coord, factor_1) == dummy_coord
```

Comment:
Looking at this for loop, I am a little confused as to why you are passing in a grid at all. Why not just have this test go:

```python

@pytest.mark.parametrize('factor', [1, 2])
@pytest.mark.parametrize('dummy_coord', [
	(0, 0),
	(0, 1),
	... # and so on
])
def test_grid_map(factor, dummy_coord):
	expected_cell = (dummy_coord[0] * factor, dummy_coord[1] * factor)
	assert grid_map(dummy_coord, factor) == expected_cell
```

The fact that the coordinates are from a grid is not actually relevant to this test.

The same applies to the test below.